contributions/catalog/controllers/config.py

```diff
@@ -1,43 +1,24 @@
 import os
+from dotenv import load_dotenv
 
 
+# Load .env file
+load_dotenv()
+
 class Config(object):
-    CONTRIBUTION_BUILDING_BLOCK_URL = os.getenv("CONTRIBUTION_BUILDING_BLOCK_URL",
-                                                "http://localhost:5000/contributions")
+    OAUTHLIB_INSECURE_TRANSPORT = bool(os.getenv('DEBUG', 'False') == 'True')
```

Comment:
Thanks for making other changes. I thought it might be better to use a separate environment variable.
```suggestion
    OAUTHLIB_INSECURE_TRANSPORT = bool(os.getenv('OAUTHLIB_INSECURE_TRANSPORT', 'False') == 'True')
```